src/component/Grid.js

```diff
@@ -0,0 +1,26 @@
+import React from 'react'
+import VideoGrid from '../component/VideoGrid';
+import '../style/Grid.css';
+
+
+class Grid extends React.Component {
+
+   render() {
+  
+      return (
+         <>
```

Comment:
@RajanTank Why these empty tags are there in all pages?

Reply:
@ParthBarot-BoTreeConsulting this is the shortcut of <React.Fragment>. as you know in react everything is tree node so we must pass the one node from the return because we can return two node 
`return(
<div/>
<div/>
)`